src/app/docs/radio/radio-docs.component.html

```diff
@@ -7,8 +7,84 @@
     The radio button module ceates a small set of selection inputs for users to select one option.
   </sky-docs-demo-page-summary>
 
+  <sky-docs-demo>
+    <sky-fluid-grid>
+      <sky-row>
+        <sky-column screenSmall="6">
+          <h4>Standard radio button</h4>
+          <sky-radio-group
+            ariaLabel="Demo for radio button component"
+            name="radioDemo"
+            value="selected"
+          >
+            <ul
+              class="sky-list-unstyled"
+            >
+              <li>
+                <sky-radio
+                  value="selected"
+                >
+                  <sky-radio-label>
+                    A selected option
+                  </sky-radio-label>
+                </sky-radio>
+              </li>
+              <li>
+                <sky-radio
+                  value="unselected"
+                >
+                  <sky-radio-label>
+                    An unselected option
+                  </sky-radio-label>
+                </sky-radio>
+              </li>
+              <li>
+                <sky-radio
+                  value="disabled"
+                  [disabled]="true"
+                >
+                  <sky-radio-label>
+                    This option cannot be selected right now
+                  </sky-radio-label>
+                </sky-radio>
+              </li>
+            </ul>
+          </sky-radio-group>
+        </sky-column>
+        <sky-column screenSmall="6">
+          <h4>Icon radio group</h4>
+          <div
+            class="sky-switch-icon-group"
+          >
+            <sky-radio
+              icon="table"
+              label="Table"
+              name="iconRadioDemo"
```

Comment:
Our docs say this property is deprecated in favore of the `name` property on `sky-radio-group`. Seems like we should either change this part of the demo to align with our recommendations in the docs or change the recommendation in the docs. Same question on the other `sky-radio` elements.

Reply:
This code won't show to the consumer, but the code-example code will! I went to double check the code example, and removed the `name` property.